src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js

```diff
@@ -41,10 +70,67 @@ class CreateBaselineModal extends Component {
         toggleCreateBaselineModal();
     }
 
-    renderModalBody() {
-        const { baselineName } = this.state;
+    renderRadioButtons() {
+        const { fromScratchChecked, copyBaselineChecked, copySystemChecked } = this.state;
+
+        return (<React.Fragment>
+            <Radio
+                isChecked={ fromScratchChecked }
+                id='create baseline'
+                label='Create baseline from scratch'
+                name='radio-1'
```

Comment:
what's the difference between name and id? does name have to be radio-1, radio-2, radio-3?

Reply:
I built it like the example from patternfly, but honestly, I think we could trash the name field. What do you think?

```
import React from 'react';
import { Radio } from '@patternfly/react-core';

class ControlledRadio extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      check1: false,
    };

    this.handleChange = (_, event) => {
      const { value } = event.currentTarget;
      this.setState({ [value]: true });
    };
  }

  render() {
    return (
      <React.Fragment>
        <Radio
          isChecked={this.state.check1}
          name="radio-1"
          onChange={this.handleChange}
          label="Controlled radio"
          id="radio-controlled"
          value="check1"
        />
      </React.Fragment>
    );
  }
}
```